docker-compose.yml

```diff
@@ -5,25 +5,15 @@ services:
   mysql:
     image: mysql:5.7.22
     restart: on-failure
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
     environment:
-      - MYSQL_HOST=${MYSQL_HOST}
-      - MYSQL_DATABASE=${MYSQL_DATABASE}
-      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
-      - MYSQL_USER=${MYSQL_USER}
-      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
-      - MYSQL_PORT=${MYSQL_PORT}
-
-      - UDW_ENDPOINT=${UDW_ENDPOINT}
-      - UDW_USER=${UDW_USER}
-      - UDW_PASSWORD=${UDW_PASSWORD}
-      - UDW_PORT=${UDW_PORT}
-      - UDW_DATABASE=${UDW_DATABASE}
-    entrypoint: ['docker-entrypoint.sh', '--default-authentication-plugin=mysql_native_password']
+      - MYSQL_ROOT_PASSWORD=student_dashboard_root_pw
```

Comment:
Do you know why this particular value was left here?

Reply:
I left it there since `MYSQL_ROOT_PASSWORD` isn't in `.env.sample` (so wouldn't be included automatically in a normal setup) and I wasn't sure if you guys would still need it or not

Reply:
[I see it in there!](https://github.com/tl-its-umich-edu/my-learning-analytics/blob/f814032134c51ad5bf6a0b99eb55c9321330a7cc/.env.sample#L36) ;) I don't see anything recent with the blame for that.

This file has a conflict too, though I didn't see what the conflict was on.

Reply:
oh right, sorry about that. Its been a while since I last looked at this

Reply:
I know, me too, sorry about that. This was going to be a pretty big change but will be great to have!

Reply:
I'll remove this env variable, rebase to fix the conflict, and take a look over to make sure there haven't been any db changes since I last looked

Reply:
alright, remove `MYSQL_ROOT_PASSWORD`, fixed conflict, and double check the models so there shouldn't be any problems atm